tests/integration/targets/ios_bgp_address_family/tasks/main.yaml

```diff
@@ -1,4 +1,8 @@
 ---
+- name: Set a fact for 'run_test'
+  ansible.builtin.set_fact:
+    run_test: false
```

Comment:
Don't skip anything, just remove or comment out l2 specific attributes that were added we can enable them once we have a l2 appliance.